magma/syntax/sequential2.py

```diff
@@ -197,6 +199,56 @@ def _seq_phi(s, t, f):
     return s.ite(t, f)
 
 
+class PrevReplacer(ast.NodeTransformer):
+    """Replace self.attr.prev() with __magma_sequential2_attr"""
+    def __init__(self, env):
+        self.env = env
```

Comment:
Looks like `env` is unused, can we remove this attribute/constructor arg?

Reply:
sure. also renamed private classes with `_` prefix.